packages/clay-card/package.json

```diff
@@ -32,6 +32,7 @@
 		"@clayui/label": "^3.3.1",
 		"@clayui/layout": "^3.1.0",
 		"@clayui/link": "^3.2.0",
+		"@clayui/shared": "^3.1.2",
```

Comment:
Looking at this better, I think it would not be very useful to have `shared` as a dependency only for noop, I think we can remove this.

Reply:
cc @diegonvs

Reply:
Sure, I'll remove